test/benchmarks/driver_bench/src/driver.mts

```diff
@@ -0,0 +1,181 @@
+
+/**
+ * The path to the MongoDB Node.js driver.
+ * This MUST be set to the directory the driver is installed in
+ * NOT the file "lib/index.js" that is the driver's export.
+ */
+export const MONGODB_DRIVER_PATH = (() => {
+  let driverPath = process.env.MONGODB_DRIVER_PATH;
+  if (!driverPath?.length) {
+    driverPath = path.resolve(__dirname, '../../../..');
+  }
+  return driverPath;
+})();
+
+/** Grab the version from the package.json */
+export const { version: MONGODB_DRIVER_VERSION } = require(
+  path.join(MONGODB_DRIVER_PATH, 'package.json')
+);
+
+/**
+ * Use git to optionally determine the git revision,
+ * but the benchmarks could be run against an npm installed version so this should be allowed to fail
+ */
+export const MONGODB_DRIVER_REVISION = (() => {
+  try {
+    return child_process
+      .execSync('git rev-parse --short HEAD', {
+        cwd: MONGODB_DRIVER_PATH,
+        encoding: 'utf8'
+      })
+      .trim();
+  } catch {
+    return 'unknown revision';
+  }
+})();
+
+/**
+ * Find the BSON dependency inside the driver PATH given and grab the version from the package.json.
+ */
+export const MONGODB_BSON_PATH = path.join(MONGODB_DRIVER_PATH, 'node_modules', 'bson');
+export const { version: MONGODB_BSON_VERSION } = require(
+  path.join(MONGODB_BSON_PATH, 'package.json')
+);
+
+/**
+ * If you need to test BSON changes, you should clone, checkout and build BSON.
+ * run: `npm link` with no arguments to register the link.
+ * Then in the driver you are testing run `npm link bson` to use your local build.
+ *
+ * This will symlink the BSON into the driver's node_modules directory. So here
+ * we can find the revision of the BSON we are testing against if .git exists.
+ */
+export const MONGODB_BSON_REVISION = await (async () => {
+  const bsonGitExists = await fs.access(path.join(MONGODB_BSON_PATH, '.git')).then(
+    () => true,
+    () => false
+  );
+  if (!bsonGitExists) {
+    return 'installed from npm';
+  }
+  try {
+    return child_process
+      .execSync('git rev-parse --short HEAD', {
+        cwd: path.join(MONGODB_BSON_PATH),
+        encoding: 'utf8'
+      })
+      .trim();
+  } catch {
+    return 'unknown revision';
+  }
+})();
+
+export const MONGODB_CLIENT_OPTIONS = (() => {
+  const optionsString = process.env.MONGODB_CLIENT_OPTIONS;
+  let options = undefined;
+  if (optionsString?.length) {
+    options = JSON.parse(optionsString);
+  }
+  return { ...options };
+})();
+
+export const MONGODB_URI = (() => {
+  if (process.env.MONGODB_URI?.length) return process.env.MONGODB_URI;
+  return 'mongodb://127.0.0.1:27017';
+})();
+
+export function snakeToCamel(name: string) {
+  return name
+    .split('_')
+    .map((s, i) => (i !== 0 ? s[0].toUpperCase() + s.slice(1) : s))
+    .join('');
+}
+
+import type mongodb from '../../../../mongodb.js';
+export type { mongodb };
+
+const { MongoClient /* GridFSBucket */ } = require(path.join(MONGODB_DRIVER_PATH));
```

Comment:
We can remove the commented import

Reply:
Instead lemme just bring over the API we're going to need for the GridFS tests, this are the only two imports we should be using for perf tests so might as well lock them in here.